src/test/compile-fail/feature-gate-negate-unsigned.rs

```diff
@@ -16,7 +16,7 @@ impl std::ops::Neg for S {
     fn neg(self) -> u32 { 0 }
 }
 
-const _MAX: usize = -1;
+const _MAX: usize = -(2 - 1);
```

Comment:
Should add a `FIXME` comment to return to the original code once it is possible (i.e. the current consteval code is removed).